Mixin tests failing

On recent PRs, the mixin tests are failing with:

```
#!/bin/bash -eo pipefail
cd doc/alertmanager-mixin; make lint
jb install
GET https://github.com/grafana/grafonnet-lib/archive/55cf4ee53ced2b6d3ce96ecce9fb813b4465be98.tar.gz 200
mixtool generate alerts mixin.libsonnet > alertmanager_alerts.yaml
find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
	while read f; do \
		jsonnetfmt -n 2 --max-blank-lines 1 --string-style s --comment-style s "$f" | diff -u "$f" -; \
	done
mixtool lint mixin.libsonnet
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel 'Alerts', target idx '0' invalid PromQL query 'sum(alertmanager_alerts{job=~"$job"}) by (job,instance)': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel 'Alerts receive rate', target idx '0' invalid PromQL query 'sum(rate(alertmanager_alerts_received_total{job=~"$job"}[$__rate_interval])) by (job,instance)': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel 'Alerts receive rate', target idx '1' invalid PromQL query 'sum(rate(alertmanager_alerts_invalid_total{job=~"$job"}[$__rate_interval])) by (job,instance)': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel '$integration: Notifications Send Rate', target idx '0' invalid PromQL query 'sum(rate(alertmanager_notifications_total{job=~"$job", integration="$integration"}[$__rate_interval])) by (integration,job,instance)': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel '$integration: Notifications Send Rate', target idx '1' invalid PromQL query 'sum(rate(alertmanager_notifications_failed_total{job=~"$job", integration="$integration"}[$__rate_interval])) by (integration,job,instance)': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel '$integration: Notification Duration', target idx '0' invalid PromQL query 'histogram_quantile(0.99,
  sum(rate(alertmanager_notification_latency_seconds_bucket{job=~"$job", integration="$integration"}[$__rate_interval])) by (le,job,instance)
) 
': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel '$integration: Notification Duration', target idx '1' invalid PromQL query 'histogram_quantile(0.50,
  sum(rate(alertmanager_notification_latency_seconds_bucket{job=~"$job", integration="$integration"}[$__rate_interval])) by (le,job,instance)
) 
': instance selector not found
[target-instance-rule] 'Alertmanager / Overview': Dashboard 'Alertmanager / Overview', panel '$integration: Notification Duration', target idx '2' invalid PromQL query 'sum(rate(alertmanager_notification_latency_seconds_sum{job=~"$job", integration="$integration"}[$__rate_interval])) by (job,instance)
/
sum(rate(alertmanager_notification_latency_seconds_count{job=~"$job", integration="$integration"}[$__rate_interval])) by (job,instance)
': instance selector not found
2022/04/29 21:25:06 failed to lint the file mixin.libsonnet: 8 lint errors found
make: *** [Makefile:17: lint] Error 1

Exited with code exit status 2

```

--- a/doc/alertmanager-mixin/.lint
+++ b/doc/alertmanager-mixin/.lint
@@ -1,7 +1,7 @@
 exclusions:
-  panel-job-instance-rule:
+  target-instance-rule:
     reason: no need to have every query contains two matchers within every selector - `{job=~"$job", instance=~"$instance"}`
-  template-job-rule: 
+  template-job-rule:
     entries:
     - dashboard: Alertmanager / Overview 
       reason: multi-select is not always required

